server/server.go

```diff
@@ -885,7 +885,7 @@ func (s *server) PauseNode(ctx context.Context, req *rpcpb.PauseNodeRequest) (*r
 		return nil, err
 	}
 
-	if err := s.network.updateNodeInfo(); err != nil {
+	if err := s.network.UpdateNodeInfo(); err != nil {
```

Comment:
Was this broken in main?

Reply:
Are these two different functions?

Reply:
Yup the upper one takes a look in network.go. By mistake the lower case one was used. The chance is to be uniform in the usage of the locked version by all server calls (AddNode, RestartNode, etc).

Reply:
Can you open a ticket to change the name of one of those methods? This seems ripe for future bugs.

Reply:
https://github.com/ava-labs/avalanche-network-runner/issues/548